crates/oxc_transformer/src/typescript/annotations.rs

```diff
@@ -543,13 +543,73 @@ impl<'a> TypeScriptAnnotations<'a> {
         statements
     }
 
+    /// ```TypeScript
+    /// import b = babel;
+    /// import AliasModule = LongNameModule;
+    ///
+    /// ```JavaScript
+    /// var b = babel;
+    /// var AliasModule = LongNameModule;
+    /// ```
+    fn transform_ts_import_equals(
+        &self,
+        decl: &mut Box<'a, TSImportEqualsDeclaration<'a>>,
+    ) -> Declaration<'a> {
+        let kind = VariableDeclarationKind::Var;
+        let decls = {
+            let binding_identifier = BindingIdentifier::new(SPAN, decl.id.name.clone());
+            let binding_pattern_kind = self.ctx.ast.binding_pattern_identifier(binding_identifier);
+            let binding = self.ctx.ast.binding_pattern(binding_pattern_kind, None, false);
+
+            let init = match &mut *decl.module_reference {
+                TSModuleReference::TypeName(type_name) => self.transform_ts_type_name(type_name),
+                TSModuleReference::ExternalModuleReference(reference) => {
+                    let callee = self.ctx.ast.identifier_reference_expression(
+                        IdentifierReference::new(SPAN, "require".into()),
+                    );
+                    let arguments = self.ctx.ast.new_vec_single(Argument::Expression(
+                        self.ctx.ast.literal_string_expression(reference.expression.clone()),
+                    ));
+                    self.ctx.ast.call_expression(SPAN, callee, arguments, false, None)
+                }
+            };
+            self.ctx.ast.new_vec_single(self.ctx.ast.variable_declarator(
+                SPAN,
+                kind,
+                binding,
+                Some(init),
+                false,
+            ))
+        };
+        let variable_declaration =
+            self.ctx.ast.variable_declaration(SPAN, kind, decls, Modifiers::empty());
+
+        Declaration::VariableDeclaration(variable_declaration)
+    }
```

Comment:
Should this be in this file? This is TypeScript transform so move this to `typescript.rs`?

Reply:
I moved this to `module.rs`